browser/src/FederationPage.tsx

```diff
@@ -0,0 +1,236 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link, PageHeading } from '@gnomad/ui'
+
+// @ts-ignore - TS2307 Cannot fine module ... or its corresponding type declarations.
```

Comment:
I'm guessing that we have to use `ts-ignore` here liberally because these are assets as opposed to code?

Reply:
Yeah exactly, these `@ts-ignore`s are currently annoyingly used in any place like this, e.g. the Stats page or Team page also where are using import statements for assets.